libs/ui/menu/brain/src/lib/brn-menu-trigger.directive.ts

```diff
@@ -13,6 +14,32 @@ export class BrnMenuTriggerDirective {
 	private readonly _cdkTrigger = inject(CdkMenuTrigger, { host: true });
 	private readonly _align = signal<BrnMenuAlign>(undefined);
 
+	/**
+	 * Observable that emits when the trigger is closed.
+	 * @type {Observable<void>}
+	 * @readonly
+	 */
+	readonly closed$: Observable<void> = this._cdkTrigger.closed.asObservable();
+
+	/**
+	 * Observable that emits when the trigger is opened.
+	 * @type {Observable<void>}
+	 * @readonly
+	 */
+	readonly opened$: Observable<void> = this._cdkTrigger.opened.asObservable(); 
+
+	/**
+	 * Event emitter that emits when the trigger is closed.
+	 * @type {EventEmitter<void>}
+	 */
+	@Output() closed = this._cdkTrigger?.closed;
```

Comment:
We should maybe use the output function or more specifically outputFromObservable here. Mind as well use the new stuff while we're here. In turn we might not need the other 2 observables opened$ and closed$ since users can also use the outputToObservable with the signal outputs. But welcome to keep them also of we feel like it might be a better a experience to just have those as pure observables as well

Reply:
Agree with you! i changed it :)